Source/bmalloc/bmalloc/DebugHeap.cpp

```diff
@@ -157,30 +156,30 @@ void* DebugHeap::memalignLarge(size_t alignment, size_t size)
 {
     alignment = roundUpToMultipleOf(m_pageSize, alignment);
     size = roundUpToMultipleOf(m_pageSize, size);
-    void* result = tryVMAllocate(alignment, size);
+    auto result = tryVMAllocateAligned(alignment, size);
```

Comment:
Discussed about it on slack. We should just use VirtualAlloc2 since Windows 8, 8.1 are EOL.